test/helpers/redis.ts

```diff
@@ -10,8 +10,15 @@ export const clearKeys = async (prefix) => {
 
 export const clearAccountLocks = () => clearKeys("locks:account")
 
-export const clearLimiters = async () => {
+export const clearLimitersWithExclusions = async (exclusions: string[]) => {
   for (const limiter in RateLimitPrefix) {
-    await clearKeys(RateLimitPrefix[limiter])
+    const limiterValue = RateLimitPrefix[limiter]
+    if (!exclusions.includes(limiterValue)) {
```

Comment:
suggestion but not require a change, whenever is possible try to not use `negated conditional`. this could be something like `if (exclusions.includes(limiterValue)) continue` (improves readability)